provision/testdata/flavor_openshift_411_baremetal.inp.yaml

```diff
@@ -0,0 +1,52 @@
+aci_config:
+  system_id: kube
+  use_legacy_kube_naming_convention: True
```

Comment:
we seem to copying over an older file; if this is the only input file we are using for testing this flavor, we should not have this configuration set (we could have variations of this file if we also want to test this configuration)

Reply:
Okay, config is removed.